docs/source/create/avatars/avatar-standards.md

```diff
@@ -164,6 +164,91 @@ Here is an example of modifying your blendshapes in your FST file:
     bs = LipsUpperClose = head_BS_mouth_down = 0.1  <br />
 </div>
 
+**ReadyPlayerMe Blendshapes**
+
+ReadyPlayerMe blendshapes are converted into Vircadia blendshapes automatically using a default mapping.
```

Comment:
```suggestion
ReadyPlayerMe blendshapes are converted into Vircadia blendshapes automatically.
```
The average user won't care how this is done, and the additional info doesn't help technical users.